### PR TITLE
fix(entities-upstreams-targets): expose tcp_failures [FTI-6057]

### PR DESCRIPTION
**Pull request by 2eha0**
# Summary

#### Resources

- [Creating a package docs](https://github.com/Kong/public-ui-components/blob/main/docs/creating-a-package.md)

### CODE REVIEW

File: packages/entities/entities-upstreams-targets/src/components/UpstreamsFormActiveHealthCheck.vue [hunk 1]
```diff
@@ -426,7 +426,10 @@ const typeItems = ref<HealthCheckTypeSelectItem[]>([
 ])
 
 const isHttps = computed((): boolean => props.type === 'https' || props.type === 'grpcs')
-const isTcp = computed((): boolean => props.type === 'tcp')
+const isTcp = computed((): boolean => {
+  console.log('isTcp:', props.type === 'tcp')
+  return props.type === 'tcp'
+})
```
Comment (Leopoldthecoder) [inline packages/entities/entities-upstreams-targets/src/components/UpstreamsFormActiveHealthCheck.vue:432]: We can revert this change.

**Review by Leopoldthecoder — APPROVED** [review]
LGTM

**Review by nekolab — APPROVED** [review]
LGTM

**Review by sumimakito — APPROVED** [review]
Manually tested and LGTM
